### ISSUE TITLE
Version change causing lag spikes.

### ISSUE THREAD

**Issue by noot200** [simibubi/Above-and-Beyond#346]
When I converted the version of my modpack from 1.1 to 1.3 it started causing lagspikes, these lagspikes occur every 5 seconds even when I'm sitting still, and disappear only when I go into the menu. The lag spikes do not improve when I change video settings or chunk configurations, and I doubt will be improved if I use better JVM arguments because it was working just fine in 1.1.

**Comment by Naralas-zz** [comment]
I am having the same issue, and after a few weird symptoms (signs that had no text showing up, gears that were frozen) and other problems, I ended up getting the same errors you do

Mine:
`01.02 09:42:09 [Server] Netty Epoll Server IO #12/ERROR Could not serialize zmaster587.advancedRocketry.command.ReflectionArgument@72682b8f (class zmaster587.advancedRocketry.command.ReflectionArgument) - will not be sent to client!`

Yours:
`[25Jan2022 10:19:01.427] [Netty Server IO #2/ERROR] [net.minecraft.command.arguments.ArgumentTypes/]: Could not serialize zmaster587.advancedRocketry.command.ReflectionArgument@7cc46d88 (class zmaster587.advancedRocketry.command.ReflectionArgument) - will not be sent to client!`

The irritating part is that the problem is degenerative and I had to roll back my server world often to even be able to join, and once I really got deeper into contraptions it became a bigger and bigger deal to roll it back. Finally I tried to clear all the entities made by AdvnacedRocketry by running the world with the mod removed entirely and it ran like a dream, but the problem is that I am not sure what kind of issues that will cause with the quest lines especially considering that AdvancedRocketry is the actual endgame.

Anyway, in short, I had the same issues, similar errors, and removing AdvancedRocketry from the server and on my end caused the entire thing to run great. In fact, I used the same world file, put advancedRocketry back in, and got a day of amazing stability and responsiveness even AFTER returning the mod, but it seems to have degenerated again to the point where I can only occasionally and after many restarts per session, even join my world.

Specifically expect the server to appear to be running, but to crash when someone actually tries to login, and for the server to complain it is running behind even with nobody playing.

AdvancedRocketry isn't playing nice is all I can say. Though I am glad to hear that you had more stability on 1.1, I might try that.
